### ISSUE TITLE
159-kodi-remote-control-app

### ISSUE THREAD

**Issue by retifrav** [retifrav/retifrav.github.io#16]
[XBMC/Kodi remote control application](https://retifrav.github.io/blog/2018/09/01/kodi-remote-control-app/) comments.

**Comment by ralts00** [comment]
Thanks for the tutorial, right now I'm going to start reading it and doing the exercise.
I know that there are many remote control apps, but none of them have the necessary well done, for example play and pause with the screen locked, enter remote control mode with gestures.
Greetings, I'll tell you how it went :)

**Comment by TassSinclair** [comment]
Nice, thanks for documenting. I notice the codebase still separates the directional operations from seek operations, did you have any luck getting them to operate dynamically based on Kodi's state?

**Comment by retifrav** [comment]
What do you mean? It's two different commands, for example `Player.Seek` and `Input.Right`, isn't it?

**Comment by TassSinclair** [comment]
That's true. I think I misinterpreted part of the blog post.

In my case I'm planning to make the arrow keys alternate between seek and directional input, depending on if the OSD is showing.

**Comment by retifrav** [comment]
Right, then yes, as I recall, you should query the player state first. In fact, I think, I was just lazy to do that and that's why I simply binded seek command to a different shortcut.
